include/astshim/PolyMap.h

```diff
@@ -134,7 +136,7 @@ friend class Object;
         return std::static_pointer_cast<PolyMap>(_copyPolymorphic());
     }
 
-    /// Get @ref PolyMap_IterInverse "IterInverse": provide an iterative inverse transformation?
+    /// Get @ref PolyMap_IterInverse "IterInverse": does this provide an iterative inverse transformation?
```

Comment:
I have used the style above for all getters and setters throughout `astshim` with a few well-motivated minor exceptions. Changing it would be a bigger job than I want to tackle right now.

Correct that they cannot be `noexcept`.